src/Symfony/Component/Serializer/Tests/Normalizer/DateTimeNormalizerTest.php

```diff
@@ -236,13 +247,25 @@ public function denormalizeUsingTimezonePassedInContextProvider()
             \DateTime::RFC3339,
         ];
     }
-
+    /**
+     * Deprecation will be removed as of 7.0, but this test case is still legit
```

Comment:
What does this mean we should do with it in 7.0?

Reply:
Keep it, but remove the `@group legacy` and `expectDeprecation`. That's because this test case triggers the deprecation before throwing the exception, as it tries the fallback path as well (and there is no way to opt-out).
In 7.0, the exception remains.

Reply:
Then let's say so in the comment, eg this?
`// TODO: remove the @group legacy and expectDeprecation in Symfony 7.0`